presenter/src/main/java/com/mashup/presenter/ui/dialog_tests/DialogComponent.kt

```diff
@@ -0,0 +1,188 @@
+package com.mashup.presenter.ui.dialog_tests
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Dialog
+import com.mashup.presenter.R
+import com.mashup.presenter.ui.common.ChinChinConfirmButton
+import com.mashup.presenter.ui.theme.Gray_300
+import com.mashup.presenter.ui.theme.Primary_1
+import com.mashup.presenter.ui.theme.White
+
+@Preview(showBackground = true)
+@Composable
+fun TasteKeywordDialogPreview() {
+    KeywordDialog(
+        keywords = listOf(
+            "싫어하는 향", "좋아하는 음식", "좋아하는 향", "좋아하는 옷 브랜드",
+            "좋아하는 꽃", "좋아하는 술"
+        )
+    )
+
+}
+
+@Composable
+fun KeywordDialog(
+    selectedKeyword: String = "",
+    keywords: List<String> = emptyList(),
+    onClick: (String) -> Unit = {},
+    onDismissRequest: () -> Unit = {},
+) {
+    Dialog(onDismissRequest = { onDismissRequest() }) {
+        Surface(
+            modifier = Modifier
+                .width(310.dp)
+                .wrapContentHeight(),
+            shape = RoundedCornerShape(8.dp),
+            color = White
+        ) {
+            KeywordContent(
+                keywordTitle = "취향 키워드",
+                selectedKeyword = selectedKeyword,
+                keywords = keywords,
+                onDismissRequest = onDismissRequest,
+                onClick = onClick
+            )
+        }
+    }
+}
+
+@Composable
+fun KeywordContent(
```

Comment:
```suggestion
private fun KeywordDialogContents(
```

처음에 state들을 이 함수가 들고있어서 이름이랑 보는데 좀 헷갈리더라구ㅜㅜ..
다이얼로그의 컨텐츠라는걸 명시해주면 더 좋을듯합니당

Reply:
넵 ! 반영했습당